src/configure/helper/LocalGitRepoHelper.ts

```diff
@@ -60,6 +47,23 @@ export class LocalGitRepoHelper {
         return deferred.promise;
     }
 
+    private static getIncreamentalFileName(fileName: string, count: number): string {
+        return fileName.substr(0, fileName.indexOf('.')).concat(` (${count})`, fileName.substr(fileName.indexOf('.')));
+    }
+
+    private gitReference: git.SimpleGit;
+    private constructor() {
+    }
+
+    public async getUsername(): Promise<string> {
+        let username = await this.gitReference.raw([
+            'config',
+            'user.name'
+        ]);
+
+        return username;
+    }
+
```

Comment:
Any specific reason for moving functions around.
Keep field declaration at top of the class
Keep constructor at the top of the class

Reply:
This file had  lot of linting error. And i was adding one function in it so though to clean this file as well.
Static functions should come before any non-static field and pub/private function.